custom/icds_reports/const.py

```diff
@@ -178,7 +178,7 @@ class MapColors(object):
 NUM_OUT_OF_SCHOOL_ADOLESCENT_GIRLS_11_14_YEARS = 'Number of out of school adolescent girls (11-14 years)'
 
 
-GOVERNANCE_API_PAGE_SIZE = 10000
+STANDARD_API_PAGE_SIZE = 10000
```

Comment:
Nit: may be good to flag that this is cas, like `CAS_API_PAGE_SIZE`, since its definitely possible other ones would be different, like in our core data APIs

Reply:
Done